task/orka-deploy/0.2/orka-deploy.yaml

```diff
@@ -0,0 +1,192 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: orka-deploy
+  labels:
+    app.kubernetes.io/version: "0.2"
+  annotations:
+    tekton.dev/categories: Deployment
+    tekton.dev/pipelines.minVersion: "0.16.0"
+    tekton.dev/tags: "orka, macstadium, deploy, build"
+    tekton.dev/displayName: "MacStadium Orka Deploy"
+    tekton.dev/platforms: "linux/amd64"
+    tekton.dev/displayName: "orka deploy"
```

Comment:
Looks like this annotation is here twice 🙂 why change the display name from `MacStadium Orka Deploy`?

Reply:
keeping it consistent with other task displayNames, but didn't remove previous :(